.devcontainer/devcontainer.json

```diff
@@ -25,6 +25,8 @@
 				"redhat.vscode-commons",
 				"ms-vscode.cpptools",
 				"ms-vscode.cmake-tools",
+				"ms-vscode.makefile-tools",
+				"Oracle.oracle-java",
```

Comment:
Is this duplicating teh functionality from the above vscode-java-pack?

Reply:
In my experience, the Oracle extension provides better functionality in vscode. The two extensions work fine together. The Oracle plugin allows quicker code navigation by enabling users to ctrl+click on a type to get to the declaration of the type instead of needing to ctrl+f for the name.  It's a minor improvement to the workflow, so I can roll this change back and enable the plugin locally for my workspaces if we're uncomfortable adding another plugin to the mix.

Reply:
I removed the addtion of Oracle.oracle-java in https://github.com/CDOT-CV/jpo-ode/pull/106/commits/2d986752ee2332abe1362c5d86dadefa717e5bbc

Reply:
Either way is fine with me, I just wasn't familiar with the Oracle plugin and any functionality it may provide.